src/themes/DarkTheme.js

```diff
@@ -67,4 +72,35 @@ export default {
   loader: {
     backgroundColor: colors.brand,
   },
+
+  dropdown: {
+    backgroundColor: colors.tuna,
+    shadowColor: shadows.shadowColor,
+    titleColor: colors.white,
+    borderColor: colors.mako,
+    borderColorLight: colors.shark,
+
+    item: {
```

Comment:
Instead of nesting this object inside of `dropdown` I would just add another one and name it `dropdownItem` as it actually is a separate component.